Projects/hCoreUI/Sources/HedvigColors.swift

```diff
@@ -106,7 +106,10 @@ public enum BrandColor { case primaryBackground(_ negative: Bool = false)
                 trait.userInterfaceStyle == .dark ? BrandColorBase.white : BrandColorBase.black
             })
         case .primaryButtonBackgroundColor: return BrandColorBase.lavender
-        case .primaryButtonTextColor: return BrandColorBase.black
+        case .primaryButtonTextColor:
+            return UIColor(dynamic: { trait -> UIColor in
+                trait.userInterfaceStyle == .dark ? BrandColorBase.white : BrandColorBase.black
```

Comment:
I think the `.primaryButtonBackgroundColor` is lavender in dark mode, and hence the color should also be black in dark mode, did you find a case where this failed? :)

Reply:
I thought this issue https://hedvig.atlassian.net/browse/APP-1101 was because of not setting the dark mode's color. I made it white and result was this below

![simulator_screenshot_FC800ADD-3825-4072-893D-5BD4C99277F3](https://user-images.githubusercontent.com/12679792/145234698-f92ce60c-7fb6-4b83-907c-76be5f2f5988.png)

Reply:
I suspect we should actually use `.secondaryButtonTextColor` on that button 😄